torchaudio/io/_stream_writer.py

```diff
@@ -0,0 +1,257 @@
+from typing import Dict, Optional
+
+import torch
+
+
+def _format_doc(**kwargs):
+    def decorator(obj):
+        obj.__doc__ = obj.__doc__.format(**kwargs)
+        return obj
+
+    return decorator
+
+
+_encoder = """The name of the encoder to be used.
+                When provided, use the specified encoder instead of the default one.
```

Comment:
This is correct. The leading spaces have to be aligned to the actual doc indentation. Otherwise `help(StreamWriter)` prints wrong spaces, though Sphinx can handle it fine.